mises/chain.json

```diff
@@ -0,0 +1,127 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "mises",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://mises.site",
+  "pretty_name": "Mises",
+  "chain_id": "mainnet",
+  "bech32_prefix": "mis",
+  "node_home": "$HOME/.misestm",
+  "daemon_name": "misestmd",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": [60, 118],
```

Comment:
the type supports only a number. Are both coinTypes are supported?

Reply:
Yes, both are supported